pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -7395,3 +7395,126 @@ func checkPipelineRunConditionStatusAndReason(t *testing.T, reconciledRun *v1bet
 		t.Errorf("Expected reason %s but was %s", conditionReason, condition.Reason)
 	}
 }
+
+func TestPropagatePipelineTaskRunSpecMetadata(t *testing.T) {
```

Comment:
Test func name is usually named as TestTheFuncTested(), maybe that is more informative ?

Reply:
think you are right on this point. a func + case name will never get us less clear.

checked this file. naming style is a bit mix, (with or without _ )
but will go with Test[TheFuncTested]_Case as for a easier reading